instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/InstrumenterBuilder.java

```diff
@@ -159,11 +159,34 @@
     return this;
   }
 
-  public InstrumenterBuilder<REQUEST, RESPONSE> setDisabled(boolean disabled) {
-    this.disabled = disabled;
+  /**
+   * Disables the {@link Instrumenter} - it will not generate any telemetry.
+   *
+   * <p>Equivalent to calling {@code .setEnabled(false)}.
+   */
+  public InstrumenterBuilder<REQUEST, RESPONSE> disable() {
+    return setEnabled(false);
+  }
```

Comment:
this looks like it's only called from a test, do you think it will be more generally useful in the future?

Reply:
Hmm, I included it because I thought that it could be a good idea to have a method that explicitly disables the instrumenter, but on the second thought, it's not that useful -- you'd probably want to disable the instrumenter based on some configuration property, in which case you'd almost always call `.setEnabled()`.
I'll remove it, we probably don't need it in the first stable release.